google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/FakeServiceBuilder.java

```diff
@@ -31,6 +31,7 @@ public class FakeServiceBuilder {
   private final List<BindableService> services = new ArrayList<>();
   private final List<ServerTransportFilter> transportFilters = new ArrayList<>();
 
+  private int serverPort = 0;
```

Comment:
you can already get this from the returned server?
https://github.com/googleapis/java-bigtable/blob/7247c326a233adbf0d1aa3025c0c25e68bc74014/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/functional/ReadRowsTest.java#L61